docs/faq.md

```diff
@@ -1,15 +1,49 @@
+Please see the **Processing Information** section for more details.
+
 #### What type of data does refine.bio support?
 
+refine.bio currently supports gene expression data, specifically genome-scale microarray and RNA-seq data.
+See our supported [microarray](https://github.com/AlexsLemonade/refinebio/blob/dev/config/supported_microarray_platforms.csv) and [RNA-seq](https://github.com/AlexsLemonade/refinebio/blob/dev/config/supported_rnaseq_platforms.txt) platforms.
+
 #### What does "corrected" metadata mean?
 
+Scientists will often use different terminology to refer to a similar sample metadata field or _key_.
+For example, `treatment` and `treatment protocol` may make reference to the same kinds of information. 
+We attempt to perform some mapping between keys to aid in searches.
+See the **refine.bio-harmonized Metadata** section for more information, including a full list of the mappings we perform.
+
 #### Why do the values differ a little bit if I download different datasets?
 
+We've prioritized keeping expression values consistent _within_ a dataset based on the samples it contains.
+Specifically, we remove any genes that are not measured in every sample in a dataset and we do that prior to performing quantile normalization.
+
+TODO: revisit when we have the QN section of transformations more fully fleshed out.
+
 #### Why do I get a limited number of genes back when I aggregate samples from different platforms?
 
+Different platforms will often measure different sets of genes. 
+These differences can be particularly pronounced when comparing older microarray platforms to more recent platforms.
+When aggregating samples, we retain _only_ the genes present in _every sample_.
+If the dataset delivered to you has fewer genes than you were expecting for that organism, it could be the result of combining multiple platforms or it may be from an older microarray platform.
+
 #### Why can't I add certain samples to my dataset?
 
+refine.bio will sometimes obtain the metadata (e.g., sample title or experimental protocol) associated with a sample but the raw or submitter processed expression data files are in a format that we can not process. 
+We do not allow you to add these samples to your dataset because we can not deliver expression values.
+
 #### How can I find out what versions of software/packages were used to process the data?
+
+Version information for the packages we think are _most important_ for data processing is available on the pop-up displayed when you click a sample's processing information link.
+
+If you require more granular package information, TODO: how to best access the info in the YAML files?
```

Comment:
The YAML file only lists the names of the packages that we want to keep track of. I think the best way to get the processing information is using the API directly, like:
- All processors: https://api.staging.refine.bio/processors/ 
- Per sample: https://api.staging.refine.bio/samples/1/ (see the `results` section`)

Right now the second API only lists the processor's ID, but we can modify it to show the full information of that processor. 

Does this make sense?